Rebase to 5.3.3+ branch + update UG Chapter 13

kmfoley opened the pull request:
This PR includes the recent updates to the 5.3.3+ branch related to Bill bugfix for windows (basically getting your PR up to date with v5.3.3+). 

The other change is I updated the WRF and CMAQ version numbers in section 13.4 as well as the name of the WRF-CMAQ tutorial.  I also removed sections 13.5 and 13.6.  Please confirm you agree with this deletion. The namelist information in 13.6 is already in your tutorial and the benchmarking data info does not apply to this release.  However we should remember to move the benchmark information into the tutorial for the v5.4 release once we have a new benchmark.   I'm not sure why we had it in this chapter rather than the tutorial in the first place.

bhutzell committed:
deleted:    UTIL/create_ebi/src_RXNSU/junit.F
 	deleted:    UTIL/create_ebi/src_RXNSU/junit.f
 	deleted:    UTIL/inline_phot_preproc/src/CSQY_DATA.F
 	deleted:    UTIL/inline_phot_preproc/src/CSQY_DATA.f
 	deleted:    UTIL/inline_phot_preproc/src/CSQY_DATA_gen.f
 	deleted:    UTIL/inline_phot_preproc/src/CSQY_DATA_old.f
	remove case dependendent file names or unneeded files

bhutzell committed:
new file:   UTIL/create_ebi/src_RXNSU/junit.F
	-restore wanted files

bhutzell committed:
modified:   UTIL/create_ebi/src_RXNSU/junit.F
	-replace with .F version

bhutzell committed:
Update junit.F

minor edit

bhutzell committed:
modified:   UTIL/create_ebi/makefile.v50XX
        -changed junit.o to depend on junit.F
	modified:   UTIL/create_ebi/scripts/bldrun_create_ebi.csh
        -changed path account for build-run script is in scripts subdirectory

bhutzell committed:
modified:   UTIL/create_ebi/scripts/bldrun_create_ebi.csh
        -change comment line on assumed path
 	modified:   UTIL/inline_phot_preproc/bldrun.inline_phot_preproc.csh
        -changed setting of CMAQ repository to relative path

kmfoley committed:
Merge pull request #171 from bhutzell/update_v5333plus

Remove files from chemistry utilities causing problems on MAC and WINDOWS systems.

Thank you @bhutzell for resolving this issue.

kmfoley committed:
PR to 5.3.3+ branch (tag CMAQv5.3.3.2_7May2022)

kmfoley committed:
Update CMAQ_UG_ch13_WRF-CMAQ.md

kmfoley committed:
Update README.md

kmfoley committed:
Update README.md

fisidi commented on DOCS/Users_Guide/CMAQ_UG_ch13_WRF-CMAQ.md:
```diff
@@ -32,41 +32,12 @@ Hemispheric WRF-CMAQ model simulation over two decades (1990−2010) shows enhan
 ## 13.4 Latest WRF-CMAQ Release
 
-The new WRF-CMAQ model is based on WRFv4.3 and CMAQv5.3.3. It supports only RRTMG radiation scheme for short wave aerosol direct effect. It uses core-shell model to perform aerosol optics calculations rather than volume mixing technique as in the previous version of the WRF-CMAQ model. 
+The new WRF-CMAQ model is based on WRFv4.4 and CMAQv5.3.3 and later. It supports only RRTMG radiation scheme for short wave aerosol direct effect. It uses core-shell model to perform aerosol optics calculations rather than volume mixing technique as in the previous version of the WRF-CMAQ model. 
```
@kmfoley be careful with this statement. I think it should read "WRFv4.4 and CMAQ5.3.3+ and later." I'm not sure v5.3.3 or any earlier versions will work with WRFv4.4

kmfoley replied:
Thanks! I made the change to say v5.3.3.3 and later and included a link to the v5.3.3.3 tag (which I will create today).

fisidi commented on UTIL/wrfcmaq_twoway_coupler/README.md:
```diff
@@ -1,10 +1,10 @@
 # WRF-CMAQ Coupler: 
 
-The new WRF-CMAQ model is based on WRF 4.3 and CMAQ 5.3.3 and supports the RRTMG radiation scheme for short wave aerosol direct effect. The new version uses a core-shell representation to perform aerosol optics calculations rather than the volume mixing technique applied in the previous versions of WRF-CMAQ. )
+The new WRF-CMAQ model is based on WRF 4.4 and CMAQ 5.3.3 (and later) and supports the RRTMG radiation scheme for short wave aerosol direct effect. The new version uses a core-shell representation to perform aerosol optics calculations rather than the volume mixing technique applied in the previous versions of WRF-CMAQ. )
```
Again "v5.3.3+" and later...

kmfoley replied:
I deleted the wrfcmaq_twoway_coupler folder.

fisidi commented on UTIL/README.md:
```diff
@@ -12,4 +12,4 @@ The CMAQ release includes several optional utilities for model developers. Chemi
 * **[inline_phot_preproc](inline_phot_preproc/README.md)**: creates photolysis reaction parameter tables for the CCTM inline photlysis module
 * **[nml](nml/README.md)**: converts chemical mechanism csv output files from chemmech to the namelist files required by the CMAQ programs
 * **[jproc](jproc/README.md)**: calculates daily look-up tables containing clear-sky photolysis rates that are used by the tabular method for calculate photolysis rates in CMAQ CCTM
-* **[wrfcmaq_twoway_coupler](wrfcmaq_twoway_coupler/README.md)**: WRF-CMAQ code to couple WRFv4.3 and CMAQv5.3.3
```
This can be completely deleted. No wrfcmaq_twoway_coupler should be mentioned or exist (I should probably delete this entire folder...).

This code is now put directly into WRF, see [PR #1609](https://github.com/wrf-model/WRF/pull/1609).

kmfoley replied:
I deleted this line and the wrfcmaq_twoway_coupler folder.

fisidi reviewed (approved):
Apart from the few comments I had, I don't see anything other outstanding issue.

Any additional information can be added in the upcoming v5.4 release.

Approved!

kmfoley committed:
Update CMAQ_UG_ch13_WRF-CMAQ.md

Clarify that the new WRF-CMAQ model is based on WRFv4.4 and CMAQv5.3.3.3 (to be tagged today).

kmfoley committed:
Update README.md

remove wrfcmaq_twoway_coupler link

kmfoley committed:
Remove wrfcmaq_twoway_coupler folder from UTILS

kmfoley committed:
Update CMAQ_UG_ch13_WRF-CMAQ.md

kmfoley commented:
Thanks for the fast review! I made the changes.
